packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -2186,7 +2191,23 @@ QUnit.module('popup options', {
         assert.equal($overlay.is(':visible'), true, 'overlay is not hidden');
 
         $(document).trigger('dxpointerdown');
-        assert.equal($overlay.is(':visible'), false, 'overlay is hidden');
+        assert.equal($overlay.is(':visible'), true, 'overlay is not hidden');
+    });
+
+    [true, false].forEach(usePopover => {
+        QUnit.test(`Popup should have correct _loopFocus option value if usePopover=${usePopover}`, function(assert) {
+            const instance = $('#lookupOptions').dxLookup({
+                usePopover,
+            }).dxLookup('instance');
+
+            openPopupWithList(instance);
+
+            const $overlay = $(`.${OVERLAY_CLASS}.${POPUP_CLASS}`);
```

Comment:
[possibly excess change]
So won't it work without adding of an overlay class?

Reply:
[dfbacbd2aa478258d57c5229411fccba0d67e7ae]